### DISCUSSION TITLE
Select pyarmor licence installation location

### DISCUSSION THREAD

**Discussion by JaviRP9** [dashingsoft/pyarmor#1941] [General]
I am using the group license for PyArmor and I’m encountering the following issue. I performed the initial registration with the provided .txt file. Then I generated the offline device regfile as indicated in the documentation:
`pyarmor reg -g 1 pyarmor-regfile-XXXX.zip`
This allowed me to use the PyArmor license correctly. Currently, it is installed in the .pyarmor directory in my user home, but I want to install it in the /opt system directory, which I’ve seen is possible according to the pyarmor documentation. I’ve tried copying the license files generated in .pyarmor to /opt/pyarmor and changing the PyArmor "Home Path" environment variable, and I even tried running commands with:
`pyarmor --home /opt/pyarmor/ gen...`
but I still can't get it to recognize the pyarmor license
How would it be possible to install the license in /opt and have pyarmor recognize it?

**Comment by jondy** [comment]
In the offline devecie, try to run `pyarmor --home /opt/pyarmor reg pyarmor-xxxx-device.1.zip`
It doesn't work just copy files to /opt/pyarmor

**Reply by JaviRP9** [reply]
Thank you very much!!
